static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter / 2;
```

Comment:
I really want to convince you that abstracting this just increases entropy and adds congitive overhead! one more go:

someone wants to update to an attribute that uses this variable. they don't recognise it, so before changing the spacing, they have to first find its definition (40 lines up the page). they then have to find all subsequent uses of it, and make sure the change they want to make doesn't break anything else using it. if it does, they probably have to undo this. 

without abstracting it, it's a simple one line edit you can be reasonably confident has no unexpected effects…
 
if that doesn't convince you, can we rename it so its value is clearer? `$content-gutter` is a bit generic?